src/test/java/com/redhat/devtools/lsp4ij/features/formatting/TypeScriptCustomLanguageServerEditorImprovementsTest.java

```diff
@@ -0,0 +1,169 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+
+package com.redhat.devtools.lsp4ij.features.formatting;
+
+import com.intellij.application.options.CodeStyle;
+import com.intellij.openapi.actionSystem.IdeActions;
+import com.intellij.openapi.editor.CaretModel;
+import com.intellij.openapi.editor.Document;
+import com.intellij.openapi.editor.Editor;
+import com.intellij.testFramework.EditorTestUtil;
+
+/**
+ * These verify that editor improvements are disabled by default in custom (i.e., non-user-defined) language servers.
+ */
+public class TypeScriptCustomLanguageServerEditorImprovementsTest extends AbstractTypeScriptEditorImprovementsTest {
```

Comment:
This just confirms that everything is disabled by default for custom/non-user-defined language server definitions.